cloudigrade/account/models.py

```diff
@@ -13,3 +14,54 @@ class Account(BaseModel):
         db_index=True
     )  # AWS Account ID
     account_arn = models.CharField(max_length=256, unique=True)  # AWS ARN
+
+
+class Instance(BaseModel):
+    """AWS EC2 Instance model."""
+
+    account = models.ForeignKey(
+        'Account',
+        on_delete=models.CASCADE,
+        db_index=True,
+        null=False,
+    )
+    ec2_instance_id = models.CharField(
+        max_length=256,
+        unique=True,
+        db_index=True,
+        null=False,
+        blank=False,
+    )
+    region = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+    )
+
+    def __repr__(self):
+        """Get repr of this Instance."""
+        return f"<Instance {self.ec2_instance_id}>"
```

Comment:
' > "

Reply:
I think it's time to install `flake8-quotes` and run it with the build. I'll do that now!